_posts/2025-06-25-cockpit-341.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Cockpit 341
+author: jelle
+date: '2025-06-25'
+tags: cockpit
+slug: cockpit-341
+category: release
+summary: 'link to podman quadlet containers'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 341:
+
+
+## services: show link to podman page for quadlets
+
+The Cockpit services page now recognizes podman quadlet's and shows a link to the cockpit podman page.
+
+The link is only shown when cockpit-podman > 107 is installed.
+
+![screenshot of show link to podman page for quadlets]({{ site.baseurl }}/images/341-services-show-link-to-podman-page-for-quadlets.png)
+
+
+## Try it out
+
+Cockpit 341 is available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+* [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/341)
+* [Cockpit Fedora 42](https://bodhi.fedoraproject.org/updates/FEDORA-2025-3bcc30f5f0)
+* [Cockpit Fedora 41](https://bodhi.fedoraproject.org/updates/FEDORA-2025-4bc30a6154)
```

Comment:
This is missing cockpit-podman? After all, that's what the only release note is about.